src/core/Settings.js

```diff
@@ -329,6 +330,9 @@ import Events from './events/Events';
  * @property {boolean} [useChangeTypeForTrackSwitch=true]
  * If this flag is set to true then dash.js will use the MSE v.2 API call "changeType()" before switching to a different track.
  * Note that some platforms might not implement the changeType functio. dash.js is checking for the availability before trying to call it.
+ * @property {boolean} [mediaSourceDurationInfinity=true]
+ * If this flag is set to true then dash.js will allow `Infinity` to be set as the MediaSource duration otherwise the duration will be set to a super large value instead of `Infinity` to allow appending segments indefinitely. 
```

Comment:
Please add the concrete value here "super large value (Math.pow(2,32))"

Reply:
Done!